BlackJackGame/Deck.cs

```diff
@@ -19,13 +19,13 @@ public Deck()
         public List<string> GenerateDeck()
         {
         
-            List<string> _completeDeck = new List<string>();
```

Comment:
Soo this `GenerateDeck()` method is basically doing what a constructor should be doing soo you can move this block of code into `Deck()` instead

Your `Deck` object should store your `completeDeck` as a class variable i.e `_completeDeck` soo it can be accessed across all the methods in your class